src/process/task.rs

```diff
@@ -75,11 +76,28 @@ impl Task {
     pub fn schedstat(&self) -> ProcResult<Schedstat> {
         Schedstat::from_reader(FileWrapper::open_at(&self.root, &self.fd, "schedstat")?)
     }
+
+    /// Thread children from `/proc/<pid>/task/<tid>/children`
+    ///
+    /// This data will be unique per task.
```

Comment:
I'd love to see these docs expanded a bit.  In particular, the `procfs(5)` man page lists some caveats that look important and I'd like to see them replicated here.